.github/goreleaser.yaml

```diff
@@ -177,3 +176,26 @@ brews:
     folder: Formula
     install: |
       bin.install "pomerium-cli"
+
+  - # Name template of the recipe
+    name: pomerium
+    # IDs of the archives to use.
+    ids:
+      - pomerium
```

Comment:
Should this be https://github.com/pomerium/pomerium/blob/42d7f5ff8fd28fc7157f4abb80c726ab1fd55b00/.github/goreleaser.yaml#L47

Reply:
No, this is creating a package for `pomerium`.  We already have `pomerium-cli` above.

Reply:
My bad. Github's weird snipping made me lose the thread.